content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/lifo.rst

```diff
@@ -0,0 +1,66 @@
+============
+LIFO removal
+============
+
+The *Last In, First Out* (LIFO) removal strategy picks the **newest** products on-hand, based on the
+date they entered a warehouse's stock.
+
+Every time an order is placed for products with the :abbr:`LIFO (Last In, First Out)` method, a
+transfer is created for the lot/serial number that has most recently entered the stock (the **last**
+lot/serial number that entered the warehouse's inventory).
+
+.. warning::
+   In many countries, the :abbr:`LIFO (Last In, First Out)` removal strategy is banned, since it can
+   potentially result in old, expired, or obsolete products being delivered to customers.
+
+Consider the following example, with the product, `Cinder Block`, which is tracked :guilabel:`By
+Lots` in the :guilabel:`Inventory` tab of the product form. Its product category's :guilabel:`Force
+Removal Strategy` is set to :guilabel:`Last In, First Out (LIFO)`.
+
+.. seealso::
+   - :ref:`Set up force removal strategy <inventory/warehouses_storage/removal-config>`
+   - :ref:`Enable lots tracking <inventory/warehouses_storage/lots-setup>`
+   - :ref:`Check arrival date <inventory/warehouses_storage/arrival_date>`
+   - :doc:`About removal strategies <removal>`
```

Comment:
I would link to the general removal strategy doc at the very beginning of each specific removal strategy doc, and then link to each of the other docs as they're mentioned. None of the information will make sense if they haven't read the general doc first, so we want to point them toward that right away.